src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -275,21 +275,33 @@
                :on-done on-done}))
 
 
-(defn- server-unavailable-dialog [dismiss]
-  (dialog/standard-dialog
-    {:width 500
-     :dismiss-self dismiss
-     :header "Server Unavailable"
-     :show-cancel? false
-     :ok-button [comps/Button {:text "OK" :onClick dismiss}]
-     :content
-     [:div {}
-      "FireCloud service is temporarily unavailable.  If this problem persists, check "
-      [:a {:href "http://status.firecloud.org/" :target "_blank"}
-       "http://status.firecloud.org/"]
-      " for more information."]}))
-
-
+(defn- show-server-unavailable-dialog [dismiss]
+  (modal/push-modal
+    (dialog/standard-dialog
+      {:width 500
+       :dismiss-self dismiss
+       :header "Server Unavailable"
+       :show-cancel? false
+       :ok-button [comps/Button {:text "OK" :onClick dismiss}]
+       :content
+       [:div {}
+          "FireCloud service is temporarily unavailable.  If this problem persists, check "
+          [:a {:href "http://status.firecloud.org/" :target "_blank"}
+           "http://status.firecloud.org/"]
+          " for more information."]})))
+
+
+  (defn- show-maintenance-mode-dialog [dismiss]
```

Comment:
This function duplicates a bit too much code from the previous. Maybe these two can be combined into:
```clojure
(defn- show-system-status-dialog [maintenance-mode? dismiss]
...)
```